get-started/get-namecoins/index.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: page
+title: How to get namecoins
+---
+
+{::options parse_block_html="true" /}
+
+Namecoins are a token-like currency used to register and update Namecoin names.  If you want to register a Namecoin name, you'll need to obtain some namecoins.
+
+## Prerequisites
+
+To get namecoins, you first need a Namecoin wallet such as Namecoin Core.  See the Downloads page.
+
+## Exchanges
+
+The easiest way to get namecoins is to purchase them on an exchange.  See the [list of exchanges]({{site.baseurl}}exchanges/).
+
+You can check current [Namecoin prices at CoinMarketCap](https://coinmarketcap.com/currencies/namecoin/#markets), current [Namecoin prices at Bitsquare](https://market.bitsquare.io/?market=nmc_btc), and [Namecoin historical price](https://bitcoinwisdom.com/markets/btce/nmcusd) charts at BitcoinWisdom. 
```

Comment:
This website promulgates the incorrect notion that cryptocurrencies have a "market cap." They do not. I would urge you to reconsider linking to them. If the idea is to show people what the current price of Namecoins is, then perhaps remove the coinmarketcap link, and leave the other two intact.